### DISCUSSION TITLE
Why choose HyperHDR

### DISCUSSION THREAD

**Discussion by reedy-au** [awawa-dev/HyperHDR#791] [General]
It's been some years and i've had 2 x 75"  Sony TVs replaced whilst running HyperHDR and it's still going solid.

I'm on v17 still... Looking to upgrade now Sony are coming to replace the panel, again, instead of the TV.

Maybe don't buy Sony.

Thanks again awawa I will upgrade soon.

Running HyperHDR on libreelec with BTC strips 144.

Also have HyperHDR running on 42 old sony no issues and samsung 98" no issues

**Comment by awawa-dev** [comment]
Hi
Great that you're happy :) Yes, in term of HyperHDR development version v17 was groundbreaking. Oh man, that was quite a few years ago. I hope you will be equally pleased with the latest release for your new setup. Good luck!
